code/modules/mob/living/living.dm

```diff
@@ -1401,14 +1401,14 @@
 				/mob/living/basic/stickman,
 				/mob/living/basic/stickman/dog,
 				/mob/living/simple_animal/crab,
+				/mob/living/basic/mushroom,
```

Comment:
keep this list alphabetical, there's extensions for vscode that will do this for you.